[Doc] Improve description of the pos parameter in gui (#1904)
taichi-dev/taichi: docs/gui.rst

Paint on a window
-----------------

> Paint simple geometric objects on a window. Taichi has internal support for lines, triangles, rectangles, circles and texts.

Reviewer: ```suggestion
Taichi has internal support for lines, triangles, rectangles, circles and texts. You can easily paint these simple geometries on the window by invoking the following APIs.
```

> Paint simple geometric objects on a window. Taichi has internal support for lines, triangles, rectangles, circles and texts.

Reviewer 2: ```suggestion
Taichi's GUI supports painting simple geometric objects, such as lines, triangles, rectangles, circles, and text.
```

.. note::

>     The position parameter ``pos`` expects an input of a tuple of 2, whose values are the relative position of the object. 

Reviewer 2: ```suggestion
    The position parameter ``pos`` expects an input of a 2-element tuple, whose values are the relative position of the object. 
```

>     ``(0.0, 0.0)`` stands for the lower left corner of the window, and ``(1.0, 1.0)`` stands for the upper right corner.

Reviewer: May also mention in `gui.get_cursor_pos()` as it follows this law too?

    :parameter gui: (GUI) the window object
    :parameter content: (str) the text to draw

>     :parameter pos: (tuple of 2, range [0,1] ) the relative position of the top-left point of the fonts / texts

Reviewer: ```suggestion
    :parameter pos: (tuple of 2, range [0,1]) the relative position of the top-left point of the fonts / texts
```

Author: I agree! I submitted a new commit with a brief introduction section. The explanation of pos goes into a Note section. Please review.
